src/index.js

```diff
@@ -1,4 +1,7 @@
+/* eslint-disable no-unused-vars */
 import './style.css';
+import interact from './interact.js';
```

Comment:
- [ ] I think you should remove line 3 because you are not using interact in this module instead of disabling the linter. Kindly make changes. :pray:

Reply:
Hi, Thank you for the feedback.
Please note that I am using the line 3. if I remove it my app will break. I wouldn't leave it if it's not needed.

Reply:
I will leave it optional but, I think this shouldn't have happened if you have followed best practices. If you don't use the `interact` var here, it shouldn't be required here. :pray: